examples/bond_linuxbridge_vlan_absent.yml

```diff
@@ -0,0 +1,20 @@
+---
+interfaces:
+  - name: eth1
+    type: ethernet
+    state: absent
```

Comment:
- There should be no need to specify the type.
- Ethernet types should use `down` as they cannot be removed.

Reply:
Changed this.